flytekit/common/translator.py

```diff
@@ -272,7 +273,9 @@ def get_serializable_node(
             inputs=entity.bindings,
             upstream_node_ids=[n.id for n in upstream_sdk_nodes],
             output_aliases=[],
-            task_node=workflow_model.TaskNode(reference_id=task_spec.template.id),
+            task_node=workflow_model.TaskNode(
+                reference_id=task_spec.template.id, overrides=TaskNodeOverrides(resources=entity._resources)
```

Comment:
These aren't the Resource models though right?  They're the internal dataclass.  Don't they need to be translated?

Reply:
they are already translated https://github.com/flyteorg/flytekit/pull/523/files#diff-b58fe30a139359c0e865665c57312b6fafd564875cd34e10f8a07189b5a90913R94

Reply:
oh i see.  can we update the type hint on line 36 of node.py then?

Reply:
ah yes, thanks for the catch